src/Tools/dotnet-trace/CommandLine/Commands/ErrorCode.cs

```diff
@@ -6,10 +6,9 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal static class ErrorCodes
     {
-        public static int SessionCreationError = 1;
-        public static int FileCreationError = 2;
-        public static int TracingError = 3;
-        public static int ArgumentError = 4;
-        public static int UnknownError = 5;
+        public const int SessionCreationError = 1;
+        public const int TracingError = 3;
```

Comment:
If we wanted to remove the error code, shall we also shift the numbers. Right now it looks like 2 is reserved from something :)